count-pos-sum-neg.py

```diff
@@ -0,0 +1,15 @@
+def count_positives_sum_negatives(arr):
+    """From list, tally positive numbers, sum the negatives.""" 
+    answer_array = []
+    positives = 0
+    sum_negs = 0
+    if arr == []:
```

Comment:
A more common way to do this is to use the falsey-ness of an empty list, and say `if not arr:`.